doc/configure

```diff
@@ -89,6 +90,18 @@ RULES = [{
 }]
 
 
+def get_args():
+    pars = argparse.ArgumentParser(description="Build documentation for Litani")
+    for arg in [{
+            "flags": ["--no-html"],
+            "action": "store_true",
+            "help": "Do not build a version of the documentation in HTML",
```

Comment:
Since you use `not` everywhere you use `args.no_html` below, maybe this is a cleaner implementation:

```python
    for arg in [{
            "flags": ["--no-html"],
            "action": "store_false",
            "help": "Do not build a version of the documentation in HTML",
            "dest": "gen_html"
    }]:
```
That way you can just jump directly to `args.gen_html` without having to use `not args.no_html` first